clap_complete/src/generator/utils.rs

```diff
@@ -1,12 +1,12 @@
 //! Helpers for writing generators
 
-use clap::{Arg, Command};
+use clap::{builder::StyledStr, Arg, Command};
```

Comment:
Should we leave this file alone and instead mov things over as needed so we have more flexibility?

Reply:
good question, on the one hand, holding them together makes it easier to keep them in line, on the otherhand, with my changes they are only a 4-liner.

Reply:
I suspect we'll be needing to make bigger changes for the filtering PR.

How about you have a commit where you fork these functions into `completer.rs` and then rebase this commit on top.